src/components/layout/Footer.tsx

```diff
@@ -0,0 +1,24 @@
+import Envelope from '@/assets/icons/envelope.svg';
+import Instagram from '@/assets/icons/instagram.svg';
+import Facebook from '@/assets/icons/facebook.svg';
+
+export default function Footer() {
+  return (
+    <div className="h-126 w-full bg-gray-10 px-20 pt-32 pb-16 font-regular text-gray-50 md:h-100 md:px-32 md:py-37 lg:flex lg:justify-center lg:px-238">
+      <div className="grid h-78 grid-cols-[1fr_auto] grid-rows-1 gap-x-59 gap-y-40 md:flex md:h-26 md:items-center md:justify-between lg:w-964">
```

Comment:
```suggestion
    <div className="h-126 w-full bg-gray-10 px-20 pt-32 pb-16 font-regular text-gray-50 md:h-100 md:px-32 md:py-37 lg:px-238">
      <div className="grid h-78 grid-cols-[1fr_auto] grid-rows-1 gap-x-59 gap-y-40 md:flex md:h-26 md:items-center md:justify-between">
```
💬 이렇게 두어도 똑같은 스타일로 보이지 않나요?